src/lib/pages/frontsite/Series/SeriesDetails/SeriesPanel/SeriesPanel.js

```diff
@@ -61,7 +61,9 @@ class SeriesPanel extends Component {
                 </Grid.Column>
                 <Grid.Column width={5}>
                   <SeriesAccess urls={series.metadata.access_urls} />
-                  <SeriesSequences relations={series.metadata.relations} />
+                  {series.metadata.relations && (
```

Comment:
isn't it the `series.metadata.relations` an Object? I think in this case it will be always evaluated as true